.travis.yml

```diff
@@ -2,6 +2,9 @@ os: linux
 dist: xenial
 language: php
 
+git:
+  autocrlf: false
```

Comment:
I had similar issues running the tests locally on windows. It would be great to have a solution which will not only work on travis.

Does https://github.com/phpstan/phpstan-src/pull/50 Work for your problem?

Reply:
I had similar issues with other repos as well, that's why my autocrlf setting on windows is set to false by default. It works very well locally too.

I believe it's possible to configure this on a project basis. Based on what I've experienced, I'd recommend setting this to false by default and switching it back(to input or true) on projects that must absolutely have CRLF.

#50 should work too, I just didn't needed it locally so I put the same setting on travis.